Patch by MSP-Greg
without the '.to_s' call, '.search_convpath' causes a silent segv...
diff --git a/spec/ruby/core/encoding/converter/search_convpath_spec.rb b/spec/ruby/core/encoding/converter/search_convpath_spec.rb
index 9e924da023..4913681bbb 100644
--- a/spec/ruby/core/encoding/converter/search_convpath_spec.rb
+++ b/spec/ruby/core/encoding/converter/search_convpath_spec.rb
@@ -26,15 +26,10 @@
     end
 
     it "raises an Encoding::ConverterNotFoundError if no conversion path exists" do
-#      lambda do
-#        Encoding::Converter.search_convpath(
-#          Encoding::ASCII_8BIT, Encoding::Emacs_Mule)
-#      end.should raise_error(Encoding::ConverterNotFoundError)
-      begin
-        Encoding::Converter.search_convpath(Encoding::ASCII_8BIT, Encoding::Emacs_Mule)
-      rescue => e
-        e.class.should == Encoding::ConverterNotFoundError
-      end
+      lambda do
+        Encoding::Converter.search_convpath(
+          Encoding::ASCII_8BIT.to_s, Encoding::Emacs_Mule.to_s)
+      end.should raise_error(Encoding::ConverterNotFoundError)
     end
   end
 end
